ui/src/app/components/help-sidebar-icons.tsx

```diff
@@ -78,6 +85,12 @@ const styles = reactStyles({
   },
 });
 
+const containerStyle: CSSProperties = {
```

Comment:
I moved the style inside the compoundIcon, since it was only used by that component. In that case, I kept the name. If you would prefer it left outside with a different name, please let me know, and I would be glad to change it.

Reply:
that plan SGTM